### PR TITLE
fix container folder issue

### LINKED ISSUE #21: "Open parent folder" didn't work as expected
It actually **opens** the file instead of opening the **folder** where the file is stored.

### PR DESCRIPTION
**Pull request by taooceros**
fix #21

### CODE REVIEW

**Comment by jjw24** [comment]
Version bump please

**Comment by taooceros** [comment]
Will do later. Afk now.

**Comment by jjw24** [comment]
1.4.3 to 1.4.7?

**Comment by taooceros** [comment]
> 1.4.3 to 1.4.7?

Isn't the current version 1.4.6?

File: plugin.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "Name": "Everything",
   "Description": "Search Everything",
   "Author": "qianlifeng,orzfly",
-  "Version": "1.4.3",
```
Comment (jjw24) [inline plugin.json]: Hmm, how come this diff is showing 3 not 6

Comment (taooceros) [reply plugin.json]: Hmm I believe I haven't fetch before creating the branch.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**